tests/transition_to_if.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+from TestHarness import Cluster, TestHelper, Utils, WalletMgr
+from TestHarness.TestHelper import AppArgs
+
+###############################################################
+# transition_to_if
+#
+# Transition to instant-finality with multiple producers (at least 4).
+#
+###############################################################
+
+
+Print=Utils.Print
+errorExit=Utils.errorExit
+
+appArgs = AppArgs()
+appArgs.add(flag="--plugin",action='append',type=str,help="Run nodes with additional plugins")
+
+args=TestHelper.parse_args({"-d","-s","--keep-logs","--dump-error-details","-v","--leave-running","--unshared"},
+                            applicationSpecificArgs=appArgs)
+pnodes=4
+delay=args.d
+topo=args.s
+debug=args.v
+prod_count = 1 # per node prod count
+total_nodes=pnodes
+dumpErrorDetails=args.dump_error_details
+
+Utils.Debug=debug
+testSuccessful=False
+
+cluster=Cluster(unshared=args.unshared, keepRunning=args.leave_running, keepLogs=args.keep_logs)
+walletMgr=WalletMgr(True)
+
+try:
+    TestHelper.printSystemInfo("BEGIN")
+
+    cluster.setWalletMgr(walletMgr)
+
+    Print(f'producing nodes: {pnodes}, topology: {topo}, delay between nodes launch: {delay} second{"s" if delay != 1 else ""}')
+
+    Print("Stand up cluster")
+    if args.plugin:
```

Comment:
Will we ever test with extra plugins? If so, add the plugins to the test command in CMakeLists.txt.